src/js/Controller/Event/lottoTicketEvent.js

```diff
@@ -0,0 +1,26 @@
+import LottoTicketView from '../../View/lottoTicketView.js';
+
+const ticket = (function () {
+  let amount = 0;
+
+  return {
+    handleClickEvent(event) {
+      if (event.pointerType === '') {
+        if (event.target.checked) {
+          LottoTicketView.showTicketUI();
+        } else {
+          LottoTicketView.hideTicketUI();
+        }
+      }
+    },
+
+    /**
+     * @param {number} value
+     */
+    set setAmount(value) {
+      amount = value;
+    },
```

Comment:
```suggestion
    set amount(value) {
      amount = value;
    },
```
이렇게 표현하는게 더 자연스러울 것 같아요!